docs/cli/graphql-transformer/directives.md

```diff
@@ -868,9 +907,37 @@ type Profile @model @auth(rules: [{allow: owner, provider: oidc, identityClaim:
 By using a configured `oidc` provider for the API, it is possible to authenticate the users against it to perform operations on the `Post` type, and `owner` authorization is also possible.
 
 
-### Combining Authorization Rules
+### Combining Multiple Authorization Rules
+
+Amplify GraphQL APIs have a primary **default** Authentication type and, optionally, additional secondary authentication types. The objects and fields in the GraphQL schema can have rules with different authorization providers assigned based on the authentication types configured in your app.
+
+One of the most common scenarios for multiple authorization rules is for combining public and private access. Take for example a blog. Blogs usually offer public access for viewing posts (guest users as well as signed in users can view them) and private access for updating or deleting posts (signed in users that created the post).
+
+Let's take a look at how we can combine public and private access to achive this:
+
+```graphql
+type Post @model
+  @auth (
+    rules: [
+      # set owner permissions on new items, allow owners ability to read, update, and delete posts
+      { allow: owner },
+
+      # allow all authenticated users to read posts
+      { allow: private, operations: [read] },
+
+      # allow all guest users (not authenticated) to read posts
+      { allow: public, operations: [read] }
+    ]
+  ) {
+  id: ID!
+  title: String
+  owner: String
+}
+```
 
-The objects and fields in the GraphQL schema can have rules with different authorization providers assigned.
+> The above schema assumes a default authentication type of `Amazon Cognito User Pools` and a secondary authentication type of `API key`.
```

Comment:
Probably don't need backticks for the Cognito bit or API Key, unless these tokens are inputs somewhere.